elm-package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "4.2.0",
+    "version": "4.2.1",
```

Comment:
Don't bump  the project version in a PR. Leave that for the maintainers. (Although we will almost certainly release a patch fix with this.)